packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -0,0 +1,88 @@
+
+**OpenID:** `https://${yourOktaOrg}/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaOrg}/.well-known/oauth-authorization-server`
+
+### Custom Authorization Server
+
+You use a Custom Authorization Server to create and apply authorization policies to secure your APIs. An access token that is minted by a Custom Authorization Server is consumed by your APIs.
+
+Okta allows you to create multiple Custom Authorization Servers within a single Okta org that you can use to protect your own resource servers. Within each authorization server, you can define your own custom OAuth 2.0 scopes, claims, and access policies to support authorization for your APIs.
+
+#### Default Custom Authorization Server
+
+Okta provides a pre-configured Custom Authorization Server called `default`. It includes a basic access policy and a rule to quickly get you started. For simple use cases, this default Custom Authorization Server is usually all that you need.
+
+To use the `default` Custom Authorization Server, use `default` as the authorization server ID:
+
+`https://${yourOktaDomian}/api/v1/authorizationServers/default`
+
+For Custom Authorization Servers that you create yourself, the `${authServerId}` is a random ID such as `aus9o8wzkhckw9TLa0h7z`.
+
+`https://${yourOktaDomain}/api/v1/authorizationServers/${authServerId}`
+
+#### Custom Authorization Server discovery endpoints
+
+The following endpoints return OIDC or OAuth 2.0 metadata related to a Custom Authorization Server. Clients can use this information to programmatically configure their interactions with Okta. Custom scopes and custom claims aren't returned.
+
+The OpenID and OAuth discovery endpoints for a Custom Authorization Server are:
+
+**OpenID:** `https://${yourOktaDomain}/oauth2/<server id>/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaDomain}/oauth2/<server id>/.well-known/oauth-authorization-server`
+
+The OpenID and OAuth discovery endpoints for the Default Authorization Server are:
+
+**OpenID:** `https://${yourOktaDomain}/oauth2/default/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaDomain}/oauth2/default/.well-known/oauth-authorization-server`
+
+## Which authorization server should you use
+
+If you are just looking to add SSO for your OIDC-based applications, you can use your Org AS. You should also use the Org AS if you want to use [OAuth2 bearer tokens with your Okta APIs](/docs/oauth-for-okta/). Only the Org Authorization Server can mint access tokens that contain Okta API scopes.
+
+If your application has requirements such as additional scopes, customizing rules for when to grant scopes, or you need additional authorization servers with different scopes and claims, then you need to [create a Custom Authorization Server](/docs/guides/customize-authz-server/overview/).
+
+The following table describes which capabilities are supported by the Default/Custom AS and which are supported by the Okta Org AS.
```

Comment:
Why `Default/Custom` ?

Reply:
We need to be explicit that Default is a type of customAS and whatever applies to customAS applies to Default as well. Main reason for the guide is to make sure customers understand default is a type of CustomAS

Reply:
I think this goes too far. We don't want to make them seem interchangeable, which this currently does.

Reply:
See my update - does this work?